### DISCUSSION TITLE
Roles a usuarios

### DISCUSSION THREAD

**Discussion by JAnthony97** [community/community#116672] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Como dar rol de administrador a un usuario?
![image](https://github.com/community/community/assets/67430566/80a9ddf2-7b10-44e2-80a3-7b4534c8c2d4)

**Comment by queenofcorgis** [comment]
Hi #116676 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
